apps/docs/.vitepress/config.ts

```diff
@@ -169,6 +169,10 @@ export default defineConfig({
                 text: 'Wallet Transferring',
                 link: '/guide/wallets/wallet-transferring',
               },
+              {
+                text: 'Setting up test wallets',
+                link: '/guide/wallets/setting-up-test-wallets',
+              },
```

Comment:
Do you think this should be under testing now?

We could still link to it from places we deem necessary.

Reply:
Good idea 7d785cc34